Inspicio/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -107,9 +107,11 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.Property<string>("Description");
 
-                    b.Property<int>("NoOfDislikes");
+                    b.Property<int>("NoOfApprovals");
```

Comment:
Looks like I misunderstood the data model - I thought each users vote (approve/reject/change) was already stored along with the users ID, so you can say which reviewer has approved/rejected/whatever. If you have that info, then you can just count the number of reviewers that have rejected something to get the number of rejections.

It looks that data isn't stored yet, and it's just an int with no way of knowing who said what. So ignore this comment until that data exists :)